src/snowplow/shareableList/shareableListEventHandler.ts

```diff
@@ -65,6 +65,9 @@ export class ShareableListEventHandler extends EventHandler {
       data: {
         shareable_list_external_id:
           data.shareable_list.shareable_list_external_id,
+        user_id: data.shareable_list.user_id
+          ? data.shareable_list.user_id
+          : undefined,
```

Comment:
will `data.shareable_list.user_id` ever be falsy? won't we always have a value?

Reply:
oh wait! is this because it's shared on the front end and sometimes this value will be falsy?

Reply:
@jpetto adding this check as in the shareable-lists-api PR I validate the userId in the snowplow events handler, and in the case (probably will never happen but better to be safe) when the userId cannot be parsed to an int, I pass `undefined` as the userId is optional on the snowplow schema.

Reply:
okay, makes sense to me!

total nit pick and not a blocker, but you could make this just `user_id: data.shareable_list.user_id ?? undefined,` 😄